src/dao/hooks/useGovernanceDAOProposals.tsx

```diff
@@ -91,7 +91,6 @@ export function useGovernanceDAOProposals(
           proxyAdmin: proposalData?.proxyAdmin ?? "",
           proxyLogic: proposalData?.proxyLogic ?? "",
           currentLogic: proposalData?.currentLogic ?? "",
-          isAdminApproved: proposalData?.isAdminApproved ?? false,
           isAdminApprovalButtonVisible: proposalData?.isAdminApprovalButtonVisible ?? false,
```

Comment:
UI state, such as visibility, should be controlled at the component level. Application level `react-query` hooks should be used to fetch/cache/return server state.